packages/node_modules/@webex/internal-plugin-team/test/integration/spec/create.js

```diff
@@ -110,9 +110,7 @@ describe('plugin-team', () => {
       }));
 
     it('decrypts the \'add\' activity appended to the general conversation after team room is created', () => webex.internal.team.createConversation(team, {displayName, participants: [kirk]}, {includeAllTeamMembers: true})
-      .then((tc) => webex.internal.conversation.get({
-        id: team.generalConversationUuid
-      }, {
+      .then((tc) => webex.internal.conversation.get(find(team.conversations.items, {id: team.generalConversationUuid}), {
```

Comment:
this was failing on master so made this fix. internal-plugin-team tests are passing now